docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,442 @@
+
+First, we join user data with cluster labels, create separate DataFrames for each cluster, generates activity histograms for each cluster, and then concatenates these histograms into a single visualization.
+
+```python
+# activity histograms by cluster
+user_df = user_df.set_index("id").join(label_df)
+
+by_cluster_data = {
+    label: user_df[user_df["cluster_label"] == label]
+    for label in np.unique(hdbscan.labels_)
+}
+
+activity_histograms = [
+    alt.Chart(user_df_part)
+    .mark_bar()
+    .encode(x=alt.X("activity", bin=True), y="count()")
+    .properties(title=f"Activity histogram for cluster {label}")
+    for label, user_df_part in by_cluster_data.items()
+]
+
+alt.hconcat(*activity_histograms)
+```
+
+![histograms of clusters](../assets/use_cases/user_acquisition_analytics/cluster_histograms.png)
+
+From our histograms, we can observe that:
+
+- cluster -1 has an unusually large number of active users (typically, there are *very* few highly active application users)
+- cluster 2 and cluster 3 users are quite similar, but low activity
+- cluster 0 has the highest proportion of medium activity users
+- cluster 1 users are moderately active, with relatively few very high activity users and very few low activity users
+
+To see the distribution of ad_creatives across different clusters, we create a DataFrame that shows each ad_creative's count value within each cluster:
+
+```python
+pd.DataFrame(user_df.groupby("cluster_label")["ad_creative"].value_counts())
+```
+
+![ad creatives distribution across clusters](../assets/use_cases/user_acquisition_analytics/ad_creatives-per-cluster.png)
+
+observations:
+
+- outliers (cluster -1, highly active users) clicked on ad_creatives from both campaigns (as expected)
+- cluster 3 clicked on only one distinct ad_creative - from the influencer based campaign
+- clusters 0 and 2 clicked on only two distinct influencer based creatives
+- cluster 1 clicked on both campaigns' ad_creatives, but more on the first (non-influencer) campaign
+
+Now, let's get some descriptive stats for our signup dates to help us interpret our clusters' behavior further. 
+
+```python
+user_df["signup_datetime"] = [
+    datetime.fromtimestamp(ts) for ts in user_df["signup_date"]
+]
+desc = user_df.groupby("cluster_label")["signup_datetime"].describe()
+for col in desc.columns:
+    if col == "count":
+        continue
+    desc[col] = [dt.date() for dt in desc[col]]
+desc
+```
+
+![signup date per cluster group](../assets/use_cases/user_acquisition_analytics/signup_date-per-cluster.png)
+
+observations...
+
+- outliers' (cluster -1) have signup dates that are scattered
+- cluster 1's signups mostly (75%) came from clicks on the first campaign's ad_creatives
+- clusters 0, 2, and 3 signed up in response to the new (influencer-augmented) campaign only
+
+
+![summary of outcomes](../assets/use_cases/user_acquisition_analytics/summary.png)
```

Comment:
@morkapronczay
This graphic is off per above error. I'll probably just omit it and create a table of outcomes instead... can include some takeaway indicators  there too.

Reply:
you probably have to recreate the table, either via a new screenshot or a new table, the summary is now fixed in the notebook

Reply:
@morkapronczay 
<img width="898" alt="Screenshot 2024-09-26 at 09 49 27" src="https://github.com/user-attachments/assets/8e42f5a3-0b3e-4494-8817-42880359acc4">
the notebook has cluster 0's signup date as "only new" 
should be "only old"

Reply:
notebook fixed, thanks for the comments.

about the first one: I see why it was misleading, I altered to charts to have uniform axis for X and Y as well and updated all descriptions of clusters to the best of my knowledge now. I think it is now a good way to present the activites and the text is a good description now of what is going on.

Reply:
@morkapronczay 
y-axis is uniform, x-axis is not. clusters 0-3 are half the width of cluster -1.